packages/@okta/vuepress-theme-default/components/TopNavigation.vue

```diff
@@ -132,7 +122,22 @@ export default {
     if(this.$themeConfig.search_url) {
       this.search_url = this.$themeConfig.search_url
     }
-  }
+  },
+
+  mounted() {
+    let coveoScript = document.createElement('script')
```

Comment:
Why is this happening here?  Can we just include the `okta_coveo_search_developer.js` file within the config.js like the other js and css files?

Reply:
This was to ensure that the entire Vue app is mounted before adding the script. Without this, some dom elements necessary for the Coveo framework initialization were missing and causing issues.